pkg/validate/secrets.go

```diff
@@ -61,14 +61,12 @@ func (s Credentials) ValidatePath(ctx context.Context) error {
 func (s Credentials) buildCredentialserences() map[string]build.BuildReason {
```

Comment:
This function name is interesting. :-)

Reply:
I would prefer to file a separate issue.

Reply:
Yeah, sure, I was only stumbling over it.